inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/config/model/instrumentation/dataproviders/DataProviderCallSettings.java

```diff
@@ -0,0 +1,172 @@
+package rocks.inspectit.oce.core.config.model.instrumentation.dataproviders;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.val;
+import org.springframework.boot.convert.ApplicationConversionService;
+import org.springframework.core.convert.ConversionException;
+import org.springframework.core.convert.ConversionService;
+import rocks.inspectit.oce.core.config.model.instrumentation.InstrumentationSettings;
+import rocks.inspectit.oce.core.config.model.validation.ViolationBuilder;
+import rocks.inspectit.oce.core.utils.AutoboxingHelper;
+import rocks.inspectit.oce.core.utils.CommonUtils;
+
+import javax.validation.constraints.NotBlank;
+import java.util.Collections;
+import java.util.Map;
+import java.util.stream.Stream;
+
+@Data
+@NoArgsConstructor
+public class DataProviderCallSettings {
+
+
+    /**
+     * The name of the provider.
+     * References the defined providers in {@link InstrumentationSettings#getDataProviders()}.
+     */
+    @NotBlank
+    private String provider;
+
+    /**
+     * Defines input values for the provider using constants, e.g. numbers or string literals.
+     * The key of the map denotes the input variable of the provider, the value is the constant value to assign.
+     * The given string literals are converted to the corresponding primitive type or directly used as a string.
+     * Constant input only works for inputs which have a primitive type, a corresponding wrapper type or are strings.
+     */
+    private Map<@NotBlank String, Object> constantInput = Collections.emptyMap();
```

Comment:
OK I see tests are clearly stating this works :+1:

Reply:
When using the inputs from this map we just have to pipe them through an ApplicationConversionService and everything will work because of spring :)
The conversion service can also be called with a generic signature as future work, so that e.g. you could pass Lists of Durations and something like this in.